### ISSUE TITLE
Not calculating some things

### ISSUE THREAD

**Issue by redmop** [Artefact2/centralcasting#1]
It isn't calculating Birth Order, Siblings, Work Attitude, Occupation Performance, and maybe others.

php 7.2 ubuntu 18.04

Here is an output (long)

```
$ ./cast pc
Player Character                                                                         (CuMod: +7, SolMod: -1, LegitMod: +0, BiMod: +10, TiMod: +0, ViMod: +0)
================================================================================================================================================================
+ Birth Events:                                                                                                                                          (100  )
|   Character Race: Human                                                                                                                                (101  )
|   Cultural Background: Civilized-Decadent                                                                                                              (102  )
|   Social Status: Destitute                                                                                                                             (103  )
|   Birth Legitimacy: Legitimate                                                                                                                         (104  )
|   Family: None known, left to fend for self                                                                                                            (106  )
|   + Siblings:                                                                                                                                          (107  )
|   |   Birth Order:                                                                                                                                     (108  )
|   Time of Birth: February the 28th at 7 pm                                                                                                             (109  )
|   + Place of Birth: Exotic location                                                                                                                    (110  )
|   |   Exotic Birth Location: Other time period (soon transported after birth)                                                                          (111  )
|   Unusual Birth:                                                                                                                                       (112  )
|   + Parents & NPCs:                                                                                                                                    (114  )
|   |   + Occupation: Head of household has one occupation                                                                                               (114A )
|   |   |   + Civilized Occupation:                                                                                                                      (423A )
|   |   |   |   Lower Class Occupation: Farmer                                                                                                           (423B )
|   |   |   |                           Serf (agricultural slave)
|   |   |   + Occupation Performance:                                                                                                                    (426  )
|   |   |   |   Work Attitude:                                                                                                                           (426A )
|   |   |   |   Work Attitude:                                                                                                                           (426A )
|   |   |   |   Achievement Level (NPCs Only): Master Craftsman                                                                                          (426B )
|   |   |   + Occupation Check:                                                                                                                          (420-4)
|   |   |   |   + Civilized Occupation:                                                                                                                  (423A )
|   |   |   |   |   Lower Class Occupation: Farmer                                                                                                       (423B )
|   |   |   |   |                           Serf (agricultural slave)
|   |   |   |   + Occupation Performance:                                                                                                                (426  )
|   |   |   |   |   Work Attitude:                                                                                                                       (426A )
|   |   |   |   |   Work Attitude:                                                                                                                       (426A )
|   |   |   |   |   Achievement Level (NPCs Only): Master Craftsman                                                                                      (426B )
|   |   + To: other parent (usually Mother)                                                                                                              (114Z )
|   |   |   + Noteworthy Item: NPC is military veteran                                                                                                   (114B )
|   |   |   |   Type of Service: *Archer (longbowmen, crossbowmen)                                                                                       (535A )
|   |   + To: other parent (usually Mother)                                                                                                              (114Z )
|   |   |   + Noteworthy Item: NPC is noted for his personality                                                                                          (114B )
|   |   |   |   + Exotic Feature (Optional):                                                                                                             (649  )
|   |   |   |   |   + Exotic Feature Category:                                                                                                           (649A )
|   |   |   |   |   |   Behavior Tag: Unique physical habit (unconscious action eg rapid blinking, twisting hair, rubbing part of face/body etc.)        (649E )
|   |   |   |   |   Behavior Tag: Unique physical habit (unconscious action eg rapid blinking, twisting hair, rubbing part of face/body etc.)            (649E )
|   |   + To: other parent (usually Mother)                                                                                                              (114Z )
|   |   |   + Noteworthy Item: NPC is very religious/evangelizing, seeks to convert others                                                               (114B )
|   |   |   |   Deity: Agricultural goddess                                                                                                              (864  )
+ Childhood Event(s), age 3 (human equivalent):                                                                                                          (200  )
|   + Significant Event of Childhood & Adolescence: Runs away from home                                                                                  (215  )
|   |   + Runaway: And lives with nonhumans                                                                                                              (215A )
|   |   |   Nonhuman: Elf                                                                                                                                (751  )
+ Adolescent Event(s), age 13 (human equivalent):                                                                                                        (200  )
|   + Significant Event of Childhood & Adolescence: Change/upheaval occurs in family                                                                     (215  )
|   |   + Family Change Event: Change social status                                                                                                      (215B )
|   |   |   Social Status: Poor                                                                                                                          (103  )
|   + Significant Event of Childhood & Adolescence: Exotic event occurs                                                                                  (215  )
|   |   + Exotic Event: Character is attacked by thugs in dark alley at night                                                                            (544  )
|   |   |               But they run away in terror as they see character's guardian ghost
|   |   |               Ghost cannot be seen/heard by friends
|   |   |   Other: Invader (foreign warrior)                                                                                                             (750  )
|   |   |   + Parents & NPCs:                                                                                                                            (114  )
|   |   |   |   + Occupation: Head of household has one occupation                                                                                       (114A )
|   |   |   |   |   + Nomad Occupation: Special occupation                                                                                               (421A )
|   |   |   |   |   |   + Special Nomad Occupation: Civilized occupation                                                                                 (421B )
|   |   |   |   |   |   |   + Civilized Occupation:                                                                                                      (423A )
|   |   |   |   |   |   |   |   Barbarian Occupation: Farmer                                                                                             (422A )
|   |   |   |   |   + Occupation Performance:                                                                                                            (426  )
|   |   |   |   |   |   Work Attitude:                                                                                                                   (426A )
|   |   |   |   |   |   Work Attitude:                                                                                                                   (426A )
|   |   |   |   |   |   Work Attitude:                                                                                                                   (426A )
|   |   |   |   |   |   Achievement Level (NPCs Only): Journeyman                                                                                        (426B )
|   |   |   |   |   + Occupation Check:                                                                                                                  (420-4)
|   |   |   |   |   |   + Nomad Occupation: Special occupation                                                                                           (421A )
|   |   |   |   |   |   |   + Special Nomad Occupation: Civilized occupation                                                                             (421B )
|   |   |   |   |   |   |   |   + Civilized Occupation:                                                                                                  (423A )
|   |   |   |   |   |   |   |   |   Barbarian Occupation: Farmer                                                                                         (422A )
|   |   |   |   |   |   + Occupation Performance:                                                                                                        (426  )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Achievement Level (NPCs Only): Journeyman                                                                                    (426B )
|   |   |   |   + Noteworthy Item: NPC originally from different culture                                                                                 (114B )
|   |   |   |   |   Cultural Background: Civilized-Decadent                                                                                              (102  )
|   + Significant Event of Childhood & Adolescence:                                                                                                      (215  )
|   |   + Significant Event of Childhood & Adolescence: Exotic event occurs                                                                              (215  )
|   |   |   + Exotic Event: Character is attacked by thugs in dark alley at night                                                                        (544  )
|   |   |   |               But they run away in terror as they see character's guardian ghost
|   |   |   |               Ghost cannot be seen/heard by friends
|   |   |   |   Other: Invader (foreign warrior)                                                                                                         (750  )
|   |   |   |   + Parents & NPCs:                                                                                                                        (114  )
|   |   |   |   |   + Occupation: Head of household has one occupation                                                                                   (114A )
|   |   |   |   |   |   + Nomad Occupation: Special occupation                                                                                           (421A )
|   |   |   |   |   |   |   + Special Nomad Occupation: Civilized occupation                                                                             (421B )
|   |   |   |   |   |   |   |   + Civilized Occupation:                                                                                                  (423A )
|   |   |   |   |   |   |   |   |   Barbarian Occupation: Farmer                                                                                         (422A )
|   |   |   |   |   |   + Occupation Performance:                                                                                                        (426  )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Achievement Level (NPCs Only): Journeyman                                                                                    (426B )
|   |   |   |   |   |   + Occupation Check:                                                                                                              (420-4)
|   |   |   |   |   |   |   + Nomad Occupation: Special occupation                                                                                       (421A )
|   |   |   |   |   |   |   |   + Special Nomad Occupation: Civilized occupation                                                                         (421B )
|   |   |   |   |   |   |   |   |   + Civilized Occupation:                                                                                              (423A )
|   |   |   |   |   |   |   |   |   |   Barbarian Occupation: Farmer                                                                                     (422A )
|   |   |   |   |   |   |   + Occupation Performance:                                                                                                    (426  )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Achievement Level (NPCs Only): Journeyman                                                                                (426B )
|   |   |   |   |   + Noteworthy Item: NPC originally from different culture                                                                             (114B )
|   |   |   |   |   |   Cultural Background: Civilized-Decadent                                                                                          (102  )
|   + Significant Event of Childhood & Adolescence:                                                                                                      (215  )
|   |   + Significant Event of Childhood & Adolescence: Change/upheaval occurs in family                                                                 (215  )
|   |   |   + Family Change Event: Change social status                                                                                                  (215B )
|   |   |   |   Social Status: Poor                                                                                                                      (103  )
|   |   + Significant Event of Childhood & Adolescence: Exotic event occurs                                                                              (215  )
|   |   |   + Exotic Event: Character is attacked by thugs in dark alley at night                                                                        (544  )
|   |   |   |               But they run away in terror as they see character's guardian ghost
|   |   |   |               Ghost cannot be seen/heard by friends
|   |   |   |   Other: Invader (foreign warrior)                                                                                                         (750  )
|   |   |   |   + Parents & NPCs:                                                                                                                        (114  )
|   |   |   |   |   + Occupation: Head of household has one occupation                                                                                   (114A )
|   |   |   |   |   |   + Nomad Occupation: Special occupation                                                                                           (421A )
|   |   |   |   |   |   |   + Special Nomad Occupation: Civilized occupation                                                                             (421B )
|   |   |   |   |   |   |   |   + Civilized Occupation:                                                                                                  (423A )
|   |   |   |   |   |   |   |   |   Barbarian Occupation: Farmer                                                                                         (422A )
|   |   |   |   |   |   + Occupation Performance:                                                                                                        (426  )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Work Attitude:                                                                                                               (426A )
|   |   |   |   |   |   |   Achievement Level (NPCs Only): Journeyman                                                                                    (426B )
|   |   |   |   |   |   + Occupation Check:                                                                                                              (420-4)
|   |   |   |   |   |   |   + Nomad Occupation: Special occupation                                                                                       (421A )
|   |   |   |   |   |   |   |   + Special Nomad Occupation: Civilized occupation                                                                         (421B )
|   |   |   |   |   |   |   |   |   + Civilized Occupation:                                                                                              (423A )
|   |   |   |   |   |   |   |   |   |   Barbarian Occupation: Farmer                                                                                     (422A )
|   |   |   |   |   |   |   + Occupation Performance:                                                                                                    (426  )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Achievement Level (NPCs Only): Journeyman                                                                                (426B )
|   |   |   |   |   + Noteworthy Item: NPC originally from different culture                                                                             (114B )
|   |   |   |   |   |   Cultural Background: Civilized-Decadent                                                                                          (102  )
|   |   + Significant Event of Childhood & Adolescence:                                                                                                  (215  )
|   |   |   + Significant Event of Childhood & Adolescence: Exotic event occurs                                                                          (215  )
|   |   |   |   + Exotic Event: Character is attacked by thugs in dark alley at night                                                                    (544  )
|   |   |   |   |               But they run away in terror as they see character's guardian ghost
|   |   |   |   |               Ghost cannot be seen/heard by friends
|   |   |   |   |   Other: Invader (foreign warrior)                                                                                                     (750  )
|   |   |   |   |   + Parents & NPCs:                                                                                                                    (114  )
|   |   |   |   |   |   + Occupation: Head of household has one occupation                                                                               (114A )
|   |   |   |   |   |   |   + Nomad Occupation: Special occupation                                                                                       (421A )
|   |   |   |   |   |   |   |   + Special Nomad Occupation: Civilized occupation                                                                         (421B )
|   |   |   |   |   |   |   |   |   + Civilized Occupation:                                                                                              (423A )
|   |   |   |   |   |   |   |   |   |   Barbarian Occupation: Farmer                                                                                     (422A )
|   |   |   |   |   |   |   + Occupation Performance:                                                                                                    (426  )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Work Attitude:                                                                                                           (426A )
|   |   |   |   |   |   |   |   Achievement Level (NPCs Only): Journeyman                                                                                (426B )
|   |   |   |   |   |   |   + Occupation Check:                                                                                                          (420-4)
|   |   |   |   |   |   |   |   + Nomad Occupation: Special occupation                                                                                   (421A )
|   |   |   |   |   |   |   |   |   + Special Nomad Occupation: Civilized occupation                                                                     (421B )
|   |   |   |   |   |   |   |   |   |   + Civilized Occupation:                                                                                          (423A )
|   |   |   |   |   |   |   |   |   |   |   Barbarian Occupation: Farmer                                                                                 (422A )
|   |   |   |   |   |   |   |   + Occupation Performance:                                                                                                (426  )
|   |   |   |   |   |   |   |   |   Work Attitude:                                                                                                       (426A )
|   |   |   |   |   |   |   |   |   Work Attitude:                                                                                                       (426A )
|   |   |   |   |   |   |   |   |   Work Attitude:                                                                                                       (426A )
|   |   |   |   |   |   |   |   |   Achievement Level (NPCs Only): Journeyman                                                                            (426B )
|   |   |   |   |   |   + Noteworthy Item: NPC originally from different culture                                                                         (114B )
|   |   |   |   |   |   |   Cultural Background: Civilized-Decadent                                                                                      (102  )
+ Adulthood Event(s):                                                                                                                                    (200  )
|   + Significant Event of Adulthood: Exotic event occurs                                                                                                (217  )
|   |   + Exotic Event: Character befriends intelligent nonhumanoid monster                                                                              (544  )
|   |   |   Monster: Evil humanoid (orc, troll, goblin, etc.)                                                                                            (756  )
+ Alignment & Attitude:                                                                                                                                  (318  )
|   Lightside Trait: Cheerful (always happy and smiling)                                                                                                 (647  )
|   Trait Strength (Optional): Weak                                                                                                                      (318D )
|   Neutral Trait: Emotionless (rarely shows emotions)                                                                                                   (318B )
|   Trait Strength (Optional): Average                                                                                                                   (318D )
|   + Personality Trait Check:                                                                                                                           (318A )
|   |   Neutral Trait: Emotionless (rarely shows emotions)                                                                                               (318B )
|   |   Trait Strength (Optional): Average                                                                                                               (318D )
|   Darkside Trait: Selfish (unwilling to share time and possessions)                                                                                    (648  )
|   Trait Strength (Optional): Average                                                                                                                   (318D )
|   + Personality Trait Check:                                                                                                                           (318A )
|   |   Darkside Trait: Selfish (unwilling to share time and possessions)                                                                                (648  )
|   |   Trait Strength (Optional): Average                                                                                                               (318D )
|   Personality Trait Check:                                                                                                                             (318A )
|   Alignment: Neutral                                                                                                                                   (318C )
Done!: Now what?                                                                                                                                         (300  )
```

**Comment by Artefact2** [comment]
I can't reproduce the issue, does it happen all the time? Do you have a way to force it to happen?

**Comment by redmop** [comment]
Every time. I'm running ubuntu under windows 10. I'll try it in normal
Ubuntu in a bit. What os/php are you using?

**Comment by redmop** [comment]
Ubuntu 19.10 real install. From default, I only added php (7.3), aptitude, and git (with dependancies). It doesn't calculate Siblings (107) (and birth order(108)), Unusual Birth(112), Rival Who (762a), Wonderful Event (529), Occupation Performance (426) Work attitude (426a), and maybe others

**Comment by Artefact2** [comment]
I can reproduce the issue in a 19.10 VM. I'm investigating the issue.

**Comment by Artefact2** [comment]
Fixed in 79f56692ec59e802bb46d52d4bde5f143aea3a5f. Thanks for reporting the bug!

**Comment by redmop** [comment]
Looks good. php.ini needs zend.assertions=1 on 18.04
